src/SUMMARY.md

```diff
@@ -4,6 +4,8 @@
 
 ## Products
 
+* [Android-l10n](products/android_l10n/README.md)
+    * [Adding new locales to specific projects](products/android_l10n/adding_new_locales.md).
```

Comment:
Might be good to shorten this one, since it's in the sidebar. "Adding new locales" could be enough?